developer/js/source/lexical-model-compiler/model-pseudoclosure.ts

```diff
@@ -0,0 +1,222 @@
+import { defaultApplyCasing, 
+         defaultCasedSearchTermToKey, 
+         defaultSearchTermToKey
+       } from "./model-defaults";
+
+/**
+ * Processes certain defined model behaviors in such a way that the needed closures
+ * may be safely compiled to a JS file and loaded within the LMLayer.
+ * 
+ * This is accomplished by writing out a 'pseudoclosure' within the model's IIFE,
+ * then used to build _actual_ closures at LMLayer load time.  This 'pseudoclosure'
+ * will very closely match the organizational patterns of this class in order to 
+ * facilitate the maintenance of this approach.
+ */
+export class ModelPseudoclosure {
+  static readonly COMPILED_NAME = 'definitions';
```

Comment:
"PSEUDOCLOSURE" keeps being used as a constant in this code. Perhaps extract constant to the file-scope, i.e.,

```suggestion
const PSEUDOCLOSURE = 'definitions';

/**
 * Processes certain defined model behaviors in such a way that the needed closures
 * may be safely compiled to a JS file and loaded within the LMLayer.
 * 
 * This is accomplished by writing out a 'pseudoclosure' within the model's IIFE,
 * then used to build _actual_ closures at LMLayer load time.  This 'pseudoclosure'
 * will very closely match the organizational patterns of this class in order to 
 * facilitate the maintenance of this approach.
 */
export class ModelPseudoclosure {
```

And then get rid of the `let PSEUDOCLOSURE` lines in the rest of the file.